client/blocks/product-purchase-features/product-purchase-features-list/video-audio-posts.jsx

```diff
@@ -16,6 +16,9 @@ export default localize( ( { selectedSite, translate } ) => {
 			<PurchaseDetail
 				icon="image-multiple"
 				title={ translate( 'Video and audio posts' ) }
+				trailing_space_test={ translate( 'Just a random string with trailing space ' ) }
```

Comment:
:information_source: Whitespace detected at the start of the string. Is this intentional? Please remember to use components rather than string concatenation.